pkg/platform/kube/getter.go

```diff
@@ -46,22 +46,27 @@ func (g *getter) get(consumer *consumer, getOptions *platform.GetOptions) ([]pla
 	functioncrInstances := []functioncr.Function{}
 
 	// if identifier specified, we need to get a single function
-	if getOptions.Name != "" {
+	if len(getOptions.MatchCriterias) > 0 {
 
-		// get specific function CR
-		function, err := consumer.functioncrClient.Get(getOptions.Namespace, getOptions.Name)
-		if err != nil {
+		// Iterate for every function name in MatchCriterias
+		for funcIndex := 0; funcIndex < len(getOptions.MatchCriterias); funcIndex++ {
```

Comment:
Use a normal `range` statement here - `for` loops aren't suitable for this situation (`for funcIndex, matchCriteria := range getOptions.MatchCriterias`)